cmd/compose/convert.go

```diff
@@ -139,15 +137,10 @@ func runConvert(ctx context.Context, streams api.Streams, backend api.Service, o
 		return nil
 	}
 
-	var out io.Writer = streams.Out()
 	if opts.Output != "" && len(content) > 0 {
-		file, err := os.Create(opts.Output)
-		if err != nil {
-			return err
-		}
-		out = bufio.NewWriter(file)
+		return os.WriteFile(opts.Output, content, 0o666)
 	}
-	_, err = fmt.Fprint(out, string(content))
+	_, err = fmt.Fprint(streams.Out(), string(content))
```

Comment:
This could even be;

```go
_, err = streams.Out().Write(content)
```

But no need to change; there's tons of places where that's not used (so should perhaps be looked at throughout the code)

Reply:
Thank you for sharing this so I can learn from it. I appreciate it.